automation/open_container_shell.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
+
+container_id="$(docker ps --format '{{.ID}} {{.Image}}' | grep nmstate-dev \
```

Comment:
I think it needs to warn or fail if there are more than one instance detected, otherwise it may be dangerous (changing the wrong container).

Reply:
Not sure why it is dangerous but I changed it to warn...